client/src/components/RecipeView.jsx

```diff
@@ -1,20 +1,12 @@
-import React from 'react';
 import Pagination from './Pagination';
 import Recipe from './Recipe';
-import '../styles/RecipeView.css';
+import '@/styles/RecipeView.css';
 import { Row, Col } from 'react-bootstrap';
 
 // Displays recipes in a grid, contains control of how many per page and pagination
 export default function RecipeView(props) {
-  const {
-    onPageChange,
-    data,
-    currentPage,
-    totalCount,
-    pageSize,
-    setPageSize,
-    setCheckedRecipe,
-  } = props;
+  const { onPageChange, data, currentPage, totalCount, pageSize, setPageSize, setCheckedRecipe } =
```

Comment:
I think the original version is more easily readable.

Reply:
I have a question regarding this. In the official React Docs, it says better to destructure props within the component parameter rather than what I'm doing. Which one should I choose?

Reply:
What you are doing *is* destructuring props. The comment is just formatting, not on destructuring props. 

For literature on the destructuring - https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

Reply:
done